odoo/addons/base/tests/test_views.py

```diff
@@ -4022,6 +4023,358 @@ def test_render_all_views(self):
             count, self.env.user.name, elapsed)
 
 
+@common.tagged('post_install', '-at_install')
+class TestInvisibleField(TransactionCaseWithUserDemo):
+    def test_uncommented_invisible_field(self):
+        only_log_modules = (
+            'base_setup',
```

Comment:
Shouldn't be better to get the odoo modules list from somewhere and then exclude from the list any one you don't want? I mean, nobody will remember to add new future modules here... 🧐

Reply:
The goal is not to add them. They will all be deleted in the coming months.

Reply:
Do you want I add a comment like
`# NEVER add new name in this list ! The new addons must add comment for all always invisible fields.`

Reply:
It would be nice to avoid confusion :dancers: